galsim/correlatednoise.py

```diff
@@ -729,27 +751,20 @@ def _get_update_rootps(self, shape, wcs):
         # Query using the rfft2/irfft2 half-sized shape (shape[0], shape[1] // 2 + 1)
         half_shape = (shape[0], shape[1] // 2 + 1)
         for rootps_array, saved_wcs in self._rootps_store:
-            if rootps_array.shape == half_shape:
-                if ( (wcs is None and saved_wcs.isPixelScale() and saved_wcs.scale == 1.) or
-                     wcs == saved_wcs ):
-                    use_stored = True
-                    rootps = rootps_array
-                    break
+            if rootps_array.shape == half_shape and wcs == saved_wcs:
+                use_stored = True
+                rootps = rootps_array
+                break
 
         # If not, draw the correlation function to the desired size and resolution, then DFT to
         # generate the required array of the square root of the power spectrum
         if use_stored is False:
-            newcf = galsim.ImageD(shape[1], shape[0]) # set the corr func to be the correct size
-            # set the wcs...
-            if wcs is None:
-                newcf.scale = 1.
-            else:
-                newcf.wcs = wcs
-            # Then draw this correlation function into an array.  If this is not done at the same
-            # wcs as the original image from which the CF derives, even if the image is rotated,
-            # then this step requires interpolation and the newcf (used to generate the PS below) is
-            # thus approximate at some level
-            newcf = self.drawImage(newcf)
+            # Draw this correlation function into an array.  If this is not done at the same wcs as
+            # the original image from which the CF derives, even if the image is rotated, then this
+            # step requires interpolation and the newcf (used to generate the PS below) is thus
+            # approximate at some level
```

Comment:
I made a slight edit here (5f8c3f2) just to the comment, but actually this got me thinking.  Need we make clear somehwere that this wcs functionality is not necessarily always something that people ought to use, even if they have images of noise that have a wcs?

The reason I pose the question is that typically (but not I guess *totally* exclusively) correlated noise derives from the focal plane detector array, and so a common use case would be:
* Get some images of correlated noise in your camera
* Build a CF
* Use this to generate simulations in GalSim with an approximately similar Gaussian noise pattern
For this, you wouldn't actually want to use the wcs either in the input or output images.  Where the wcs stuff is massively useful is of course when you want to whiten noise in `RealGalaxy` images that have a funky wcs relative to the COSMOS one... But that should be our problem more than the users.

Is this caveat utilitor, or should we mention it?